include/drjit-core/half.h

```diff
@@ -0,0 +1,114 @@
+/*
+    drjit-core/half.h -- minimal half precision number type
+
+    All rights reserved. Use of this source code is governed by a BSD-style
+    license that can be found in the LICENSE file.
+*/
+
+#pragma once
+
+#include <drjit-core/jit.h>
+#include <drjit-core/traits.h>
+
+#include <cmath>
+#include <cstring>
+
+#if defined __aarch64__
+#include <arm_fp16.h>
+#elif defined(__F16C__)
+#include <emmintrin.h>
```

Comment:
I think you can remove `<emmintrin.h>` here. AFAIK `<immintrin.h>` includes everything.